tests/unit/modules/test_blazeMeterUploader.py

```diff
@@ -591,6 +591,129 @@ def test_engine_metrics_send_error(self, mock_client_class):
         self.assertEqual(client.send_engine_metrics.call_count, 2)
         client.disconnect.assert_called_once()
 
+    @patch('bzt.modules.blazemeter.blazemeter_reporter.HappysocksClient')
+    def test_engine_metrics_persistent_connect_error(self, mock_client_class):
+        # error on every connect
+        # prepare mocks
+        client = mock_client_class.return_value
+        client.connected.return_value = False
+        client.connect.side_effect = TaurusNetworkError('Test error')
+        # perform test
+        reporter = BlazeMeterUploader()
+        reporter.engine = EngineEmul()
+        reporter.parameters['signature'] = '123'
+        reporter.parameters['session-id'] = 'r-v4-5f50153f49a13'
+        reporter.parameters['master-id'] = 122362
+        reporter.parameters['calibration-id'] = 4306
+        reporter.parameters['calibration-step-id'] = 10
```

Comment:
why do we still have these params?

Reply:
These were optional parameters to allow reporting calibration step id to happysocks and stored as metadata with metrics. If we no longer want this mechanism, we can have it removed.